src/components/LeftSection/directPlaylist.js

```diff
@@ -1,10 +1,23 @@
-import {status} from './library'
-
-export function DirectPlaylist({playlist, states, chooses, status}){
-  console.log(status)
+export function DirectPlaylist({data}){
   return(
-    <>
-    {status}
-    </>
-  )
+        <a href={data.link}>
+          <li className='direct-playlist'>
```

Comment:
Now go even deeper. Import here the styles connected with this component only. In this case - only those CSS classes. Later on we'll remove css files and import styles here to show you different styling methods